### ISSUE TITLE
Competency Framework description set missing connections from Learning Opportunities

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/CredentialRegistry#650]
This competency framework:
https://credentialengineregistry.org/resources/ce-24f4f084-7953-4a6c-93df-abf4841e0453

Has connections from something like two thousand learning opportunities, such as:
https://credentialengineregistry.org/resources/ce-6a233982-a3fc-4c1b-8385-b0a7b410b6a9
https://credentialengineregistry.org/resources/ce-df09e2ef-dca8-465e-b8e3-a37d9a15c7ac
https://credentialengineregistry.org/resources/ce-e81115e1-1182-4705-a84a-bc80c027210d
and many more.

However, the description set for the framework doesn't seem to include those connections.  Did the process lock up again? If so, is it due to the volume of data?
![image](https://github.com/CredentialEngine/CredentialRegistry/assets/21346519/3f0b7d35-cbf4-4d22-8bde-ff2086923582)

**Comment by excelsior** [comment]
@siuc-nate You're right, the number of the connections is the problem. I pre-calculated this description set manually, now it looks good, all of the 2,000+ competencies are included. I'll try to find a way to optimize the query to handle such large sets. 

I haven't found other missing description sets yet, If you encounter anything, let me know, I'll fix them manually in the meantime.

**Comment by siuc-nate** [comment]
Thanks - the competencies are there now, but the connections to other things (like learning opportunities that reference the competencies) are still missing.

**Comment by jeannekitchens** [comment]
@excelsior @siuc-nate please follow up on this issue.

**Comment by siuc-nate** [comment]
The description set has improved, but is still missing data for this path:
```
< ceasn:isPartOf < ceasn:Competency < ceterms:targetNode < ceterms:CredentialAlignmentObject < ceterms:teaches < ceterms:LearningOpportunityProfile
```

It could possibly be because ceterms:Course isn't getting included for the ceterms:LearningOpportunityProfile path?

**Comment by excelsior** [comment]
@siuc-nate I added the missing subclasses and now the description set for the above path looks good.

**Comment by siuc-nate** [comment]
Looks good to me, thanks.
